ddtrace/contrib/flask/patch.py

```diff
@@ -273,7 +273,9 @@ def traced_start_response(status_code, headers):
         if not span.get_tag(FLASK_ENDPOINT) and not span.get_tag(FLASK_URL_RULE):
             span.resource = u" ".join((request.method, code))
 
-        trace_utils.set_http_meta(span, config.flask, status_code=code, response_headers=headers)
+        trace_utils.set_http_meta(
+            span, config.flask, status_code=code, response_headers=headers, request_cookies=request.cookies
```

Comment:
This is the one where we set the responses related addresses, we only need the other one I think